webapp/phpunit.xml.dist

```diff
@@ -11,10 +11,6 @@
         <ini name="error_reporting" value="-1" />
         <server name="APP_ENV" value="test" force="true" />
         <server name="SHELL_VERBOSITY" value="-1" />
-        <server name="SYMFONY_PHPUNIT_REMOVE" value="" />
-        <server name="SYMFONY_PHPUNIT_VERSION" value="6.5" />
-        <ini name="date.timezone" value="UTC" />
```

Comment:
Don't we need the timezone anymore? I remember we needed it for some tests if we ran it on calca

Reply:
This was what `auto` happened, I see no problem in setting this explicit so I'll add it back if this will be the final PR